server/embed/etcd.go

```diff
@@ -307,6 +309,12 @@ func print(lg *zap.Logger, ec Config, sc config.ServerConfig, memberInitialized
 	if quota == 0 {
 		quota = storage.DefaultQuotaBytes
 	}
+	var autoCompactRetention zapcore.Field
+	if sc.AutoCompactionMode == compactor.ModePeriodic {
+		autoCompactRetention = zap.Duration("auto-compaction-retention", sc.AutoCompactionRetention)
+	} else {
+		autoCompactRetention = zap.Int64("auto-compaction-retention", int64(sc.AutoCompactionRetention))
```

Comment:
I like this simplification for less confusion. But should we keep the name the name not to breaking existing users?

Reply:
@siyuanfoundation I made the changes. Can you take a look again?